intl/msg_hash_ja.h

```diff
@@ -2026,67 +2026,67 @@ MSG_HASH(MENU_ENUM_SUBLABEL_CHEEVOS_TEST_UNOFFICIAL,
 MSG_HASH(MENU_ENUM_SUBLABEL_CHEEVOS_HARDCORE_MODE_ENABLE,
       "ステートセーブやチート、巻き戻し、早送り、一時停止、スローモーションをすべてのゲームで無効する。")
 MSG_HASH(MENU_ENUM_SUBLABEL_DRIVER_SETTINGS,
-      "システムのドライバを変更する。")
+      "システムのドライバを変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_RETRO_ACHIEVEMENTS_SETTINGS,
-      "実績に関係する設定を変更する。")
+      "実績に関係する設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_CORE_SETTINGS,
-      "コアの設定を変更する。")
+      "コアの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_RECORDING_SETTINGS,
-      "録画の設定を変更する。")
+      "録画の設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_ONSCREEN_DISPLAY_SETTINGS,
-      "OSDオーバーレイ、キーボードオーバーレイ、OSDメッセージの設定を変更する。")
+      "OSDオーバーレイやキーボードオーバーレイ、\nOSDメッセージの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_FRAME_THROTTLE_SETTINGS,
-      "巻き戻しや早送り、スローモーションの設定を変更する。")
+      "巻き戻しや早送り、スローモーションの設定を\n変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_SAVING_SETTINGS,
-      "保存に関係する設定を変更する。")
+      "保存に関係する設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_LOGGING_SETTINGS,
-      "ログの設定を変更する。")
+      "ログの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_USER_INTERFACE_SETTINGS,
-      "ユーザーインタフェースの設定を変更する。")
+      "ユーザーインターフェースの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_USER_SETTINGS,
-      "アカウントやユーザー名、言語を変更する。")
+      "アカウントやユーザー名、ユーザーインターフェースの\n言語を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_PRIVACY_SETTINGS,
-      "プライバシー設定を変更する。")
+      "プライバシーの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_DIRECTORY_SETTINGS,
-      "このシステムの初期ディレクトリーを変更する。")
+      "システムの動作に関係するファイルが格納される\n初期ディレクトリを変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_PLAYLIST_SETTINGS,
-      "プレイリストの設定を変更する。")
+      "プレイリストの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_NETWORK_SETTINGS,
-      "ネットワークの設定を変更する。")
+      "ネットワークの設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_ADD_CONTENT_LIST,
-      "コンテンツをスキャンしてデータベースに登録する。")
+      "コンテンツをスキャンしてデータベースに登録します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_AUDIO_SETTINGS,
-      "オーディオ出力の設定を変更する。")
+      "オーディオ出力の設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_BLUETOOTH_ENABLE,
       "Bluetoothを有効する。")
 MSG_HASH(MENU_ENUM_SUBLABEL_CONFIG_SAVE_ON_EXIT,
       "終了時に設定を自動的に保存する。")
 MSG_HASH(MENU_ENUM_SUBLABEL_CONFIGURATION_SETTINGS,
-      "コンフィグファイルのデフォルト設定を変更する。")
+      "システム設定ファイルのデフォルト設定を変更します。")
 MSG_HASH(MENU_ENUM_SUBLABEL_CONFIGURATIONS_LIST,
-      "コンフィグファイルを管理と作成する。")
+      "システム設定ファイルの作成と管理を行います。")
```

Comment:
The config files for RA do not affect the user's entire "system", does it make sense to use that word here?